components/UserAvatar.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <img
+    :class="[
+      'flex-shrink-0',
+      'overflow-hidden',
+      'box-border',
+      'text-light-gray',
+      'bg-current',
+      'border-[1px]',
+      'border-current',
+      'rounded-[50%]',
+    ]"
+    :src="imageSrc"
+    :style="imageStyle"
+    :loading="loadingType"
+  />
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+import { DEFAULT_AVATAR } from '../constant'
+
+@Component
+export default class UserAvatar extends Vue {
+  @Prop({ default: 40 }) readonly size: any | undefined
+  @Prop(String) readonly url: string | undefined
+
+  get imageSrc() {
+    return this.getLikeCoResizedImageUrl(this.url || DEFAULT_AVATAR, this.size)
```

Comment:
Should use DiceBear for avatar?

Reply:
For LikerId, all liker Id avatar url returns `DEFAULT_AVATAR` by default now, so `DEFAULT_AVATAR` is not needed.